### PR TITLE
[Docs] Migration docs 1st complete

### PR DESCRIPTION
**Pull request by kkdg**
### Related issues

closes #

### Which environment this relates to
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog

- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing VS sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and currently important rules acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by pkarw** [comment]
@andrzejewsky this is ok to merge

File: docs/guide/cookbook/theme.md [hunk 1]
```diff
@@ -1118,10 +1118,53 @@ var dSearchSearch = Diff2Html.getPrettyHtml(
...
+  {inputFormat: 'diff', showFiles: false, matching: 'none', outputFormat: 'line-by-line'}
+);
+document.getElementById("d-side-sidebar").innerHTML = dSideSidebar;
+</script>
+
+ _CategoryLink_ gets help from `helper` for formatting. 
+
+ _Disabling Scroll_ feature is added to _SidebarMenu_ too.
+
+- Go to `./src/themes/degi/components/core/blocks/SearchPanel/SearchPanel.vue` and add it as follows :
```
Comment (ktopouzi) [inline docs/guide/cookbook/theme.md]: I believe this link should be `/SubCategory.vue` and not `SearchPanel` again :smile:

Comment (kkdg) [reply docs/guide/cookbook/theme.md]: Oh something else already changed? let me figure it out first and fix it.

Comment (kkdg) [reply docs/guide/cookbook/theme.md]: Aha, just a plain copy/paste mistake :)
By the way, you are such an eagle eye!

Comment (ktopouzi) [reply docs/guide/cookbook/theme.md]: No problem :) I actually followed this guide step by step so I got a very close look. This rocks by the way. Thank you for the great work! :rocket: :100:

**Pull request merged**